### PR TITLE
Add option to disable fileserver in XAPI conf

### PR DESCRIPTION
**Pull request by benjamreis**
See: https://github.com/xapi-project/xen-api/pull/5608

### CODE REVIEW

**Comment by benjamreis** [comment]
[Koji scratch build towards v8.3-incoming](http://koji.xcp-ng.org/taskinfo?taskID=53645)

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -73,6 +73,7 @@ Patch1007: xapi-23.31.0-xapi-service-depends-on-systemd-tmpfiles-setup.patch
 Patch1008: xapi-23.31.0-use-lib-guess-content-type.XCP-ng.patch
 # Remove when we get 24.14.0 release
 Patch1009: xapi-24.11.0-pci-passthrough.XCP-ng.patch
+Patch1010: xapi-24.11.0-disable-fileserver-option.XCP-ng.patch
```
Comment (stormi) [inline SPECS/xapi.spec:76]: Better also add a comment stating in which release we can expect this patch in upstream XAPI

Comment (benjamreis) [reply SPECS/xapi.spec:76]: It's the same release as above 24.14 so ithought it applieed to the whole batch until there's another comment :thinking:

Comment (stormi) [reply SPECS/xapi.spec:76]: Oh, right. All good then.

**Review by stormi — APPROVED** [review]

**Comment by benjamreis** [comment]
[Koji build towards v8.3-incoming](https://github.com/xcp-ng-rpms/xapi/pull/65)

**Pull request merged**